### PR TITLE
Recut - update REAMDE, remove unnecessary files, use GitHub release workflowRecut release

### PR DESCRIPTION
**Pull request by alisnwu**

### CODE REVIEW

File: requirements/README.txt [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# YOU MAY DELETE THIS FILE AFTER SETTING UP DEPENDENCIES!
```
Comment (sbillinge) [inline requirements/README.txt:1]: please delete this file.

**Review by sbillinge — COMMENTED** [review]
please see inline.

**Comment by bobleesj** [comment]
@alisnwu  `doc/source/api/diffpy.utils.example_package.rst` file could be also removed.

File: requirements/README.txt (hunk 1 above)
Comment (alisnwu) [reply requirements/README.txt]: Sorry about that, just deleted

**Comment by bobleesj** [comment]
You can ping @sbillinge  when you are ready for review

**Comment by alisnwu** [comment]
@sbillinge ready for review

**Pull request merged**